app/controllers/supplementary/ProcedureCodesPageController.scala

```diff
@@ -188,7 +188,6 @@ class ProcedureCodesPageController @Inject()(
             }
         }
     }
-  //scalastyle:on methodLength
```

Comment:
If you remove turn on scalastyle you need to remove turn off too

Reply:
Yes I made this by _"mistake"_ let me put this back on, but the next time I see this code I am going to refactor the methods/functions to not be so long. :)